.github/workflows/conformance-ipsec-e2e.yaml

```diff
@@ -92,8 +92,8 @@ jobs:
             tunnel: 'vxlan'
             encryption: 'ipsec'
             encryption-node: 'false'
-            key-one: 'gcm(aes)'
-            key-two: 'cbc(aes)'
+            key-one: 'gcm-aes'
+            key-two: 'hmac-sha256'
```

Comment:
What's going on here?

Reply:
These are [algorithm names](https://github.com/cilium/cilium-cli/blob/main/encrypt/ipsec_key_rotator.go#L8-L12) used in Cilium CLI.

Reply:
Hm, I guess it's not worth than before :shrug:

Reply:
Yeah but without this change we will need additional mapping, isn't it?

Reply:
Sorry, I meant to write *worse* not *worth* :facepalm: 

More specifically what I mean is that users couldn't really chose the authentication algorithm before and now they can't chose the encryption algorithm (outside of the two we list). Probably doesn't matter much given we wouldn't recommend other algorithms anyway.